firewall/handlers/blacklist_handler.go

```diff
@@ -24,45 +24,43 @@ func(handler * BlacklistHandler) CheckQuery(query string) error {
 		}
 	}
 
-
 	//Check tables
 	if len(handler.tables) != 0 {
 		parsedQuery, err := sqlparser.Parse(query)
 		if err != nil {
-			return err
+			return errors.New(err.Error() + " | blacklist (tables)")
 		}
 
 		switch parsedQuery := parsedQuery.(type) {
 		case *sqlparser.Select:
 			for _, forbiddenTable := range handler.tables {
 				for _, table := range parsedQuery.From{
 					if strings.EqualFold(sqlparser.String(table.(*sqlparser.AliasedTableExpr).Expr), forbiddenTable) {
-						return ErrAccessToForbiddenTable
+						return ErrAccessToForbiddenTableBlacklist
 					}
 				}
 			}
 
 		case *sqlparser.Insert:
 			for _, forbiddenTable := range handler.tables {
 				if strings.EqualFold(parsedQuery.Table.Name.String(), forbiddenTable) {
-					return ErrAccessToForbiddenTable
+					return ErrAccessToForbiddenTableBlacklist
 				}
 			}
 
 		case *sqlparser.Update:
-
+			return errors.New("not implemented yet")
```

Comment:
add new error like `ErrNotImplemented` and then in a future we can use it in another handlers

Reply:
Ok